.vscode/settings.json

```diff
@@ -3,9 +3,6 @@
     "files.associations": {
         "*.cnxml": "xml"
     },
-    "xml.catalogs": [
```

Comment:
This was needed before to fix the XML error that would occur when first loading POET. Is this not needed anymore?

Reply:
Nope -- starting with [this PR](https://github.com/openstax/poet/pull/85) we actually tell the XML server about the catalog location through the API. That allows us to remove it here and also avoids a race condition that can happen in certain scenarios.

Reply:
I see. I was mostly being curious ;)